whois-update/src/main/java/net/ripe/db/whois/update/keycert/X509CertificateWrapper.java

```diff
@@ -75,21 +76,17 @@ private String convertFromRfc2253ToCompatFormat(String name) {
 
     @Override
     public boolean equals(final Object o) {
-        if (this == o) {
-            return true;
-        }
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
 
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
+        final X509CertificateWrapper that = (X509CertificateWrapper) o;
 
-        return ((X509CertificateWrapper) o).getCertificate().equals(
-                this.getCertificate());
+        return Objects.equals(certificate, that.certificate);
```

Comment:
I dropped this, to be consistent with the other equals() methods. I think it's clear from the context what it is.